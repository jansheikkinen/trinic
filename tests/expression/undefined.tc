// Undefined

function main(void) -> void do
  undefined;
end
