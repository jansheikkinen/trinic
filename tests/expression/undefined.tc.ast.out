## PARSER ##
Parsing tests/expression/undefined.tc...

### LEXER ###
Generating tokens from source...
Generated 11 tokens
000:     FUNCTION,     (null), (003, 008)
001:   IDENTIFIER,       main, (003, 013)
002:            (,     (null), (003, 014)
003:         VOID,     (null), (003, 018)
004:            ),     (null), (003, 019)
005:           ->,     (null), (003, 021)
006:         VOID,     (null), (003, 026)
007:           DO,     (null), (003, 029)
008:    UNDEFINED,     (null), (004, 011)
009:            ;,     (null), (004, 012)
010:          END,     (null), (005, 003)

### AST GENERATION ###
function main(VOID) -> VOID do
  UNDEFINED
  end

